package.json

```diff
@@ -1,22 +1,22 @@
 {
   "name": "get-port-cli",
-  "version": "1.1.0",
+  "version": "2.0.0",
   "description": "Get an available port",
   "license": "MIT",
   "repository": "sindresorhus/get-port-cli",
   "author": {
     "name": "Sindre Sorhus",
     "email": "sindresorhus@gmail.com",
-    "url": "sindresorhus.com"
+    "url": "https://sindresorhus.com/"
```

Comment:
Don't do unrelated changes.

Reply:
yes, thought about it but couldn't hold myself back, cause my editor was keep warning about it 😞